ecl/eclcmd/queries/ecl-queries.cpp

```diff
@@ -368,34 +398,16 @@ class EclCmdQueryFiles : public EclCmdCommon
         if (ret == 0)
         {
             IArrayOf<IConstFileUsedByQuery> &files = resp->getFiles();
-            if (!files.length())
-                fputs("No files used.\n", stdout);
+            IArrayOf<IConstQueryFilesUsed> &queries = resp->getQueries();
```

Comment:
question: how about moving this line before line 406 and use the queries in line 406 and 408? Or remove this line?